### PR TITLE
coded initial website design

### LINKED ISSUE #3: First prototype of user-facing landing page

### PR DESCRIPTION
**Pull request by lukebrdn**
Resolved #3

### CODE REVIEW

**Review by GrahamLitten — COMMENTED** [review]
im not mad, im disappointed

File: index.html [hunk 1]
```diff
@@ -1,24 +1,97 @@
...
+        Top teams get free drinks. Everyone else wins the priceless confirmation
+        they posses average intelligence, just like they feared.
+      </p>
+    </section>
+
+    <section class="flex-column">
+      <h2>Contact</h2>
+      <p>
+        If you own a bar and want us to bring our games to you, email us at
+        <a href="mailto:grahamlitten@gmail.com ">grahamlitten@gmail.com </a>.
```
[the remark this answers is not in the document]
Comment (GrahamLitten) [reply index.html:81]: confirmed -- eviltrivia@gmail.com

**Review by GrahamLitten — APPROVED** [review]

**Review by amschel-de-r — COMMENTED** [review]
No comments

**Comment by GrahamLitten** [comment]
Im not sure how to use github fully, but I believe you're looking at an outdated file. I changed the lemon images to new assets. I think. Lol.

**Comment by amschel-de-r** [comment]
Oh yeah I was just trying to remove an old notification sorry

**Pull request merged**